install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -91,39 +87,36 @@ spec:
         server: example.com
         path: "/nfsexport"
 ----
-
 ====
 
-All containers in the above pod will be members of the group 1234 and
-will have access to the volume regardless of what the container
-runs as.
-
-This is useful with volumes which do not allow you to change the
-export's ownership on the client such as NFS and GlusterFS.
+All containers in the above pod will be members of the group *1234* and will
+have access to the volume regardless of what the container runs as. This is
```

Comment:
s/regardless of what the container runs as/regardless of what user the container runs as